doc/advanced-configuration/outgoing-connections.md

```diff
@@ -205,13 +205,18 @@ Below is a sample configuration:
 
 `ConnectionOptions` can take the following `{key, value}` pairs:
 
-* `{server, HostName}` - string, default: `"http://localhost"` - the URL of the destination HTTP server (including a port number if needed).
+* `{http_client, Client}` - `gun` or `fusco`, defaults to `fusco`.
+This choice influences some of the options below.
+* `{server, Host}` - for Fusco a string, default: `"http://localhost"` - the URL of the destination HTTP server (including a port number if needed).
+For Gun a tuple `{Host, Port}`.
```

Comment:
This split is just complicated for now reason. Can we just parse URL and construct this tuple + options ourselves?

Reply:
(Again, dropping `fusco` would be even better here)